content/articles/2018-11-parsnip-0-0-1.Rmarkdown

```diff
@@ -124,6 +128,8 @@ lm_car_model
 
 There are no additional arguments that we should specify here, so let's jump to fitting the actual model. Our two choices at this point are whether to use `fit()` or `fit_xy()`. `fit()` takes a formula, while `fit_xy()` takes objects for the predictors and outcome(s). Recall that `glm` and `lm` only allow for formulas, while `glmnet` only takes a matrix of predictors and an outcome. `parsnip` allows for either so that you can avoid having to think about what the underlying model function requires. To demonstrate, let's make a simple model:
 
+AH: perhaps comment on whether `fit_xy` needs matrices, or if it can also deal with data.frame/tibble
```

Comment:
I want to stay away from adding a lot of detail in this post. To keep it concise, I've excluded a fair amount of low-level information.